### PR TITLE
Feature/eco 161 campaign manager

### PR DESCRIPTION
**Pull request by ievgenvarava**
Implemented rendering of campaigns on search, product and cart pages.

### CODE REVIEW

File: src/SprykerEco/Client/FactFinderSdk/Business/Api/Converter/RecommendationResponseConverter.php [hunk 1]
```diff
@@ -21,11 +21,18 @@ class RecommendationResponseConverter extends BaseConverter
     protected $recommendationAdapter;
 
     /**
+     * @var \SprykerEco\Client\FactFinderSdk\FactFinderSdkConfig
+     */
+    private $factFinderSdkConfig;
```
Comment (dereuromark) [inline src/SprykerEco/Client/FactFinderSdk/Business/Api/Converter/RecommendationResponseConverter.php]: We recommend to stick to protected here.

File: src/SprykerEco/Client/FactFinderSdk/FactFinderSdkClient.php [hunk 2]
```diff
@@ -87,6 +88,42 @@ public function getRecommendations(FactFinderSdkRecommendationRequestTransfer $f
     }
 
     /**
+     * Specification:
```
Comment (dereuromark) [inline src/SprykerEco/Client/FactFinderSdk/FactFinderSdkClient.php]: The specification is usually on the interface

File: src/SprykerEco/Client/FactFinderSdk/Business/Api/Converter/RecommendationResponseConverter.php (hunk 1 above)
Comment (ghost) [reply src/SprykerEco/Client/FactFinderSdk/Business/Api/Converter/RecommendationResponseConverter.php]: Fixed this one.

File: src/SprykerEco/Client/FactFinderSdk/FactFinderSdkClient.php (hunk 2 above)
Comment (ghost) [reply src/SprykerEco/Client/FactFinderSdk/FactFinderSdkClient.php]: Fixed.

File: src/SprykerEco/Zed/FactFinderSdk/Business/FactFinderSdkBusinessFactory.php [hunk 3]
```diff
@@ -65,6 +64,14 @@ public function getMoneyFacade()
     }
 
     /**
+     * @return \SprykerEco\Zed\FactFinderSdk\Business\Writer\AbstractFileWriter
+     */
+    protected function createFileWriter()
```
Comment (dereuromark) [inline src/SprykerEco/Zed/FactFinderSdk/Business/FactFinderSdkBusinessFactory.php]: createCsvFileWriter maybe?

Comment (ghost) [reply src/SprykerEco/Zed/FactFinderSdk/Business/FactFinderSdkBusinessFactory.php]: Yes, looks good

**Pull request merged**